src/libs/actions/BankAccounts.js

```diff
@@ -537,9 +537,10 @@ function setFreePlanVerifiedBankAccountID(bankAccountID) {
  * Show error modal and optionally a specific error message
  *
  * @param {String} errorModalMessage The error message to be displayed in the modal's body.
+ * @param {Boolean} isErrorModalMessageHtml if @errorModalMessage is in html format or not
  */
-function showBankAccountErrorModal(errorModalMessage = null) {
-    Onyx.merge(ONYXKEYS.REIMBURSEMENT_ACCOUNT, {isErrorModalVisible: true, errorModalMessage});
+function showBankAccountErrorModal(errorModalMessage = null, isErrorModalMessageHtml = false) {
+    Onyx.merge(ONYXKEYS.REIMBURSEMENT_ACCOUNT, {isErrorModalVisible: true, errorModalMessage, isErrorModalMessageHtml});
```

Comment:
Just to confirm here, if we call `merge` the value of `isErrorModalMessageHtml` will be overwritten right?

Reply:
I understand that is the case